src/util/labelledmatrix.jl

```diff
@@ -0,0 +1,57 @@
+export readlsm
+
+const _default_delims = [' ','\t']
+
+"""
+Reads a Labelled Square Matrix
+
+Returns a tuple of `(matrix, labels)`
+"""
```

Comment:
Docstrings in Bio.jl should follow the [guideline](http://julia.readthedocs.io/en/latest/manual/documentation/#documentation) of Julia.